src/plugins/kibana/public/settings/sections/indices/index.html

```diff
@@ -13,6 +13,13 @@
       <div>
         Pick this option if you already have data in Elasticsearch.
       </div>
+
+      <h4>
+        <a href="#/settings/indices/create/filebeat">Tail a File</a>
+      </h4>
+      <div>
+        Pick this option if you have log data you'd like to send to Elasticsearch.
```

Comment:
Log file data sounds ok to me. I wouldn't worry too much about the descriptive text, I'm guessing some wordsmith will refine these later on.